docs/source/guides/using_dask_entitysets.rst

```diff
@@ -1,8 +1,11 @@
-DFS with Dask EntitySets (BETA)
-===============================
-Support for Dask EntitySets is still in Beta - if you encounter any errors using this approach, please let us know by creating a `new issue on Github <https://github.com/FeatureLabs/featuretools/issues>`_.
+Using Dask EntitySets (BETA)
```

Comment:
can we add a link to this page at the bottom of https://docs.featuretools.com/en/dask-docs-update/loading_data/using_entitysets.html?